opm/material/fluidsystems/blackoilpvt/BrineCo2Pvt.hpp

```diff
@@ -435,7 +435,8 @@ class BrineCo2Pvt
                                                     salinity_[regionIdx],
                                                     /*knownPhaseIdx=*/-1,
                                                     xlCO2,
-                                                    xgH2O);
+                                                    xgH2O,
+                                                    extrapolate);
```

Comment:
Mmh, isn't this new parameter unused and will just cause a warning? Did you miss something or does this fix compilation issues? Please clarify.

Anyway, please remove the new warning either by using the parameter or `[[maybe_unused]]`

Reply:
I don't understand. Extrapolate which is true in BrineCo2Pvt will be passed to BinaryCoeffBrineCO2::calculateMoleFractions and overwrite the default i.e. extrapolate=false in that method.

Reply:
:flushed: Should learn how to read code. Sorry, I misread it.